packages/prop-house-webapp/src/utils/getWinningIds.ts

```diff
@@ -12,21 +12,13 @@ const getWinningIds = (
   )
     return;
```

Comment:
Should return the same type on both cases. So here we could return an empty array?

Reply:
returning empty array vs returning nothing & therefore can remove some conditionals in the UI

https://github.com/Prop-House/prop-house-monorepo/pull/344/commits/893a9643011aad2ead350701f4f04b0d7728dd67